release_notes/ose_3_1_release_notes.adoc

```diff
@@ -175,4 +258,12 @@ Features Support Scope]
 
 The following features are in Technology Preview:
 
-- Binary builds, and the `*Dockerfile*` source type for builds.
+* Binary builds, and the Dockerfile source type for builds.
+* Horizontally scaling application instances. OpenShift compares pod CPU usage and will scale appropriately depending on the `*TargetCPUConsumption*` parameter on the pod.
```

Comment:
@DirectXMan12 Ah ok. I just had this info from a line in another document. I've fixed as suggested. Thanks!